rotkehlchen/tests/integration/test_transactions.py

```diff
@@ -1,23 +1,31 @@
+from typing import TYPE_CHECKING
+
 import pytest
 from flaky import flaky
 
 from rotkehlchen.db.evmtx import DBEvmTx
 from rotkehlchen.db.filtering import EvmTransactionsFilterQuery
 from rotkehlchen.tests.utils.ethereum import (
-    ETHERSCAN_AND_INFURA_PARAMS,
+    TEST_ADDR1,
+    TEST_ADDR2,
     setup_ethereum_transactions_test,
 )
 
+if TYPE_CHECKING:
+    from rotkehlchen.chain.ethereum.transactions import EthereumTransactions
+    from rotkehlchen.db.dbhandler import DBHandler
+
 
 @flaky(max_runs=3, min_passes=1)  # failed in a flaky way sometimes in the CI due to etherscan
-@pytest.mark.parametrize(*ETHERSCAN_AND_INFURA_PARAMS)
+@pytest.mark.parametrize('ethereum_accounts', [[TEST_ADDR1, TEST_ADDR2]])
 @pytest.mark.parametrize('transaction_already_queried', [True, False])
+@pytest.mark.freeze_time('2023-01-24 22:45:45 GMT')
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
```

Comment:
what is this?

Reply:
is an option by vcr to remove a param from the query, in this case the etherscan apiKey query param

Reply:
To not record it or replay it? I guess its' nice to have but not really important as the key is already hard-coded and free :P